linux_os/guide/services/ntp/service_chronyd_or_ntpd_enabled/rule.yml

```diff
@@ -12,7 +12,11 @@ description: |-
     Note: The <tt>ntpd</tt> daemon is not enabled by default. Though as mentioned
     in the previous sections in certain environments the <tt>ntpd</tt> daemon might
     be preferred to be used rather than the <tt>chronyd</tt> one. Refer to:
-    {{{ weblink(link="https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/ch-Configuring_NTP_Using_the_chrony_Suite.html") }}}
+    {{% if product == "ol7" %}}
+        {{{ weblink(link="https://docs.oracle.com/cd/E52668_01/E54669/html/ol7-nettime.html") }}}
+    {{% else %}}
+        {{{ weblink(link="https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/ch-Configuring_NTP_Using_the_chrony_Suite.html") }}}
+    {{% endif %}}
```

Comment:
Opened https://github.com/ComplianceAsCode/content/issues/3557 to add a fedora specific conditional/link at some point.